components/org.wso2.carbon.identity.user.export.core/src/main/java/org/wso2/carbon/identity/user/export/core/internal/UserProfileExportDataHolder.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.user.export.core.internal;
+
+import org.wso2.carbon.consent.mgt.core.ConsentManager;
+import org.wso2.carbon.identity.user.export.core.service.UserInformationProvider;
+import org.wso2.carbon.identity.user.profile.mgt.association.federation.FederatedAssociationManager;
+import org.wso2.carbon.user.core.service.RealmService;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Data holder for User Profile Export service.
+ */
+public class UserProfileExportDataHolder {
+
+    private static RealmService realmService;
+    private static FederatedAssociationManager federatedAssociationManager;
+    private static ConsentManager consentManager;
+
+    public static List<UserInformationProvider> getUserInformationProviders() {
+
+        return userInformationProviders;
+    }
+
+    public static void setUserInformationProviders(
+            List<UserInformationProvider> userInformationProviders) {
+
+        UserProfileExportDataHolder.userInformationProviders = userInformationProviders;
+    }
+
+    private static List<UserInformationProvider> userInformationProviders = new ArrayList<>();
```

Comment:
Shall we add method comment for these also. There are some other method missing the method comments